ts/features/itwallet/presentation/details/components/ItwPresentationCredentialCardFlipButton.tsx

```diff
@@ -15,35 +17,55 @@ type ItwPresentationCredentialCardFlipButtonProps = {
 const ItwPresentationCredentialCardFlipButton = ({
   isFlipped,
   handleOnPress,
-  fullScreen = false
-}: ItwPresentationCredentialCardFlipButtonProps) => (
-  <View
-    style={fullScreen ? styles.fullWidthButton : styles.button}
-    accessible={true}
-    accessibilityLabel={I18n.t(
-      "features.itWallet.presentation.credentialDetails.card.showBack"
-    )}
-    accessibilityRole="switch"
-    accessibilityState={{ checked: isFlipped }}
-  >
-    <IOButton
-      variant={fullScreen ? "solid" : "link"}
-      label={I18n.t(
-        `features.itWallet.presentation.credentialDetails.card.${
-          isFlipped ? "showFront" : "showBack"
-        }`
+  fullScreen = false,
+  isL3Enabled = false
+}: ItwPresentationCredentialCardFlipButtonProps) => {
+  const rowStyle = fullScreen
+    ? styles.fullWidthButton
+    : isL3Enabled
+    ? styles.l3Button
+    : styles.button;
+
+  const shouldRenderItwBadge = !fullScreen && isL3Enabled;
```

Comment:
I'm not convinced `ItwPresentationCredentialCardFlipButton` should contain logic to display the ITW badge. I think it should be handled one level up, in the parent component.